packages/bff/src/routers/attributeRouter.ts

```diff
@@ -1,30 +1,135 @@
 import { ZodiosEndpointDefinitions } from "@zodios/core";
 import { ZodiosRouter } from "@zodios/express";
+import { bffApi } from "pagopa-interop-api-clients";
 import {
   ExpressContext,
   ZodiosContext,
   zodiosValidationErrorToApiProblem,
 } from "pagopa-interop-commons";
-import { bffApi } from "pagopa-interop-api-clients";
+import { makeApiProblem } from "../model/domain/errors.js";
+import { PagoPAInteropBeClients } from "../providers/clientProvider.js";
+import { fromBffAppContext } from "../utilities/context.js";
+import { attributeServiceBuilder } from "../services/attributeService.js";
+import { emptyErrorMapper } from "../utilities/errorMappers.js";
 
 const attributeRouter = (
-  ctx: ZodiosContext
+  ctx: ZodiosContext,
+  { attributeProcessClient }: PagoPAInteropBeClients
 ): ZodiosRouter<ZodiosEndpointDefinitions, ExpressContext> => {
   const attributeRouter = ctx.router(bffApi.attributesApi.api, {
     validationErrorHandler: zodiosValidationErrorToApiProblem,
   });
 
+  const attributeService = attributeServiceBuilder(attributeProcessClient);
+
   attributeRouter
-    .post("/certifiedAttributes", async (_req, res) => res.status(501).send())
-    .post("/verifiedAttributes", async (_req, res) => res.status(501).send())
-    .post("/declaredAttributes", async (_req, res) => res.status(501).send())
-    .get("/attributes", async (_req, res) => res.status(501).send())
-    .get("/attributes/:attributeId", async (_req, res) =>
-      res.status(501).send()
-    )
-    .get("/attributes/origin/:origin/code/:code", async (_req, res) =>
-      res.status(501).send()
-    );
+    .post("/certifiedAttributes", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+
+      try {
+        const result = await attributeService.createCertifiedAttribute(
+          req.body,
+          ctx
+        );
+
+        return res.status(200).json(result).end();
+      } catch (error) {
+        const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
```

Comment:
Since [this PR](https://github.com/pagopa/interop-be-monorepo/pull/745) we are able to add the logging here [as we already do in Scala](https://github.com/pagopa/interop-be-backend-for-frontend/blob/0d5b42922e9c49f5e0ef4d4e8ef368025e516637/src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AttributesApiServiceImpl.scala#L77); can you add these logs in these endpoints?

Reply:
Yes, done!